scripts/cibuild

```diff
@@ -25,15 +25,15 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
     else
         ./scripts/test
 
+        # Build static asset bundle for React frontend
+        docker-compose \
```

Comment:
I don't see how the bundle produced by this is making its way into the Django container image. I think that we need to add a step that takes the output of `dist` on the local file system and slurps it up into the appropriate directory in the Django container image.

```bash
# After running `cibuild`
vagrant@vagrant:/vagrant$ docker run --rm -ti --entrypoint /bin/bash open-apparel-registry-api:f7b7318
root@b34ab15ecb65:/usr/local/src# ls -l static/
total 32
drwxr-xr-x 1 root root  4096 Jan 22 21:40 admin
drwxr-xr-x 1 root root  4096 Jan 22 21:40 django_extensions
drwxr-xr-x 1 root root  4096 Jan 22 21:40 gis
drwxr-xr-x 1 root root  4096 Jan 22 21:40 rest_framework
-rw-r--r-- 1 root root 13937 Jan 23 16:10 staticfiles.json
root@b34ab15ecb65:/usr/local/src# exit
exit
vagrant@vagrant:/vagrant$ ls -l dist/
total 40
-rw-r--r-- 1 vagrant vagrant   912 Jan 23 16:11 asset-manifest.json
-rw-r--r-- 1 vagrant vagrant 23762 Jan 23 16:10 favicon.ico
-rw-r--r-- 1 vagrant vagrant  1396 Jan 23 16:11 index.html
-rw-r--r-- 1 vagrant vagrant   317 Jan 23 16:10 manifest.json
-rw-r--r-- 1 vagrant vagrant  3846 Jan 23 16:11 service-worker.js
drwxr-xr-x 1 vagrant vagrant   160 Jan 23 16:11 static
```

I could see this happening by adding a `COPY` instruction to the Django container image that always adds the contents of `dist` (`dist` may need to be moved so that it is accessible by the build context of the Django container image).

Reply:
I modified the frontend volume mount so that the static bundle is placed in `src/django/static`:

```bash
# After running `cibuild`
vagrant@vagrant:/vagrant$ docker run --rm -ti --entrypoint /bin/bash open-apparel-registry-api:874fc53
root@eeb940a622bf:/usr/local/src# ls -l static/
total 76
drwxr-xr-x 6 root root  4096 Jan 23 17:34 admin
-rw-r--r-- 1 root root   912 Jan 23 17:34 asset-manifest.json
drwxr-xr-x 5 root root  4096 Jan 23 17:34 django_extensions
-rw-r--r-- 1 root root 23762 Jan 23 17:33 favicon.ico
drwxr-xr-x 5 root root  4096 Jan 23 17:34 gis
-rw-r--r-- 1 root root  1396 Jan 23 17:34 index.html
-rw-r--r-- 1 root root   317 Jan 23 17:33 manifest.json
drwxr-xr-x 7 root root  4096 Jan 23 17:34 rest_framework
-rw-r--r-- 1 root root  3846 Jan 23 17:34 service-worker.js
drwxr-xr-x 5 root root  4096 Jan 23 17:34 static
-rw-r--r-- 1 root root 13937 Jan 23 17:34 staticfiles.json
root@eeb940a622bf:/usr/local/src# exit
vagrant@vagrant:/vagrant$ ls -l src/django/static/
total 40
-rw-r--r-- 1 vagrant vagrant   912 Jan 23 17:34 asset-manifest.json
-rw-r--r-- 1 vagrant vagrant 23762 Jan 23 17:33 favicon.ico
-rw-r--r-- 1 vagrant vagrant  1396 Jan 23 17:34 index.html
-rw-r--r-- 1 vagrant vagrant   317 Jan 23 17:33 manifest.json
-rw-r--r-- 1 vagrant vagrant  3846 Jan 23 17:34 service-worker.js
drwxr-xr-x 1 vagrant vagrant   160 Jan 23 17:34 static
vagrant@vagrant:/vagrant$
```

It looks like everything's good now. The only downside with this approach is the container build will still pass even if the static asset bundle is missing.

Reply:
This looks good. Now we're getting both the React bundle and the Django static assets served by Django under `/static`.